llvm/lib/Target/X86/X86InstrInfo.h

```diff
@@ -150,6 +150,11 @@ class X86InstrInfo final : public X86GenInstrInfo {
 public:
   explicit X86InstrInfo(X86Subtarget &STI);
 
+  const TargetRegisterClass *
```

Comment:
Add doxygen for it.

Reply:
It's an overloaded function, and was already doxygened in TargetInstrInfo.h.  Do you want me to copy the comments?

Reply:
Other overloaded functions have their comments too, it's not always identical to the base class.

Reply:
Make sense. Done.